bloom/commands/git/import_upstream.py

```diff
@@ -38,6 +38,7 @@
 import tarfile
 import tempfile
 
+from io import open
```

Comment:
I can switch these up to use `io.open`. The big downside to using io.open everywhere is that in python2 the IO object wants `unicode`s not strings whereas it's perfectly happy with strings in python3 and even if we wanted to just throw in utf-8 bytes anyways the `unicode()` cast doesn't exist for py3 and nor is decode() defined for strings.

I think this means that unless we resolve this a completely different way, every time we read or write a file with `io.open` we'll need a python version check.